csharp/test/Drivers/Snowflake/Resources/snowflakeconfig.json

```diff
@@ -4,19 +4,28 @@
     "account": "",
     "host": "",
     "database": "",
-    "user": "",
-    "password": "",
     "warehouse": "",
-    "authenticationType": "",
-    "authenticationTokenPath": "",
-    "pkcs8Passcode": "",
-    "useHighPrecision":  true,
+    "useHighPrecision": true,
     "metadata": {
         "catalog": "",
         "schema": "",
         "table": "",
         "expectedColumnCount": 0
     },
+    "authentication": {
+        "auth_oauth": {
+            "token": ""
+        },
+        "auth_jwt": {
+            "private_key_file": "",
+            "private_key_pwd": "",
+            "user": ""
+        },
+        "default": {
```

Comment:
"default" is not an auth type. This should be `auth_snowflake`

Reply:
Changed